ckks/bootstrapping/bootstrapping.go

```diff
@@ -8,47 +8,79 @@ import (
 	"github.com/tuneinsight/lattigo/v4/rlwe"
 )
 
-// Bootstrap re-encrypts a ciphertext at lvl Q0 to a ciphertext at MaxLevel-k where k is the depth of the bootstrapping circuit.
+// Bootstrap re-encrypt a ciphertext at lvl Q0 to a ciphertext at MaxLevel-k where k is the depth of the bootstrapping circuit.
 // If the input ciphertext level is zero, the input scale must be an exact power of two smaller or equal to round(Q0/2^{10}).
```

Comment:
This condition should probably be explicitly checked in the function (and then the comment should be removed). I am not sure what 2^10 means here exactly? It's probably assumed `2^{10} > ||m||^{canonical}_{\infty}` or something like that?

Reply:
Yes you are correct, and it is also not clear. I will update the godoc of the method.